model/jpa/src/main/resources/META-INF/jpa-changelog-14.0.0.xml

```diff
@@ -84,6 +85,18 @@
         </modifySql>
     </changeSet>
 
+    <changeSet author="keycloak" id="14.0.0-KEYCLOAK-18286-supported-dbs-edb-migration">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
+            <dbms type="postgresql"/>
+            <changeSetExecuted id="14.0.0-KEYCLOAK-18286-supported-dbs" author="keycloak" changeLogFile="META-INF/jpa-changelog-14.0.0.xml" />
```

Comment:
This seems to be a duplicate to `<changeSet author="keycloak" id="24.0.0-26618-edb-migration">`